pkg/network/nmstate.go

```diff
@@ -0,0 +1,40 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+func changeSafeNMState(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
+	if prev.NMState != nil && !reflect.DeepEqual(prev.NMState, next.NMState) {
+		return []error{errors.Errorf("cannot modify NMState Daemon configuration once it is deployed")}
+	}
+	return nil
+}
+
+// renderNMState generates the manifests of NMState handler
+func renderNMState(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string, clusterInfo *ClusterInfo) ([]*unstructured.Unstructured, error) {
+	if conf.LinuxBridge == nil {
```

Comment:
copy/paste issue? should be `conf.NMState`

Reply:
yes, woops.  will fix.